introduction.md

```diff
@@ -1 +1,6 @@
-This is a forked git repository. My favorite codeschool course is Javascript Roadtrip.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?